extension/src/popup/components/manageAssets/ManageAssetRows/index.tsx

```diff
@@ -138,6 +143,14 @@ export const ManageAssetRows = ({
             networkDetails,
           }),
         );
+        trackChangeTrustline();
+        dispatch(resetSubmission());
+        navigateTo(ROUTES.account);
+      }
+
+      if (submitFreighterTransaction.rejected.match(submitResp)) {
+        setErrorAsset(assetSubmitting);
+        navigateTo(ROUTES.trustlineError);
       }
```

Comment:
Handling a trustline error fell off in this release. Restoring it and refactor a bit to track properly